### ISSUE TITLE
Issue with Google Play Services 8.1.0 (java.lang.IncompatibleClassChangeError)

### ISSUE THREAD

**Issue by ultraon** [mrmans0n/smart-location-lib#87]
Hi, i use SmartLocationLib v3.1.11 (with v3.0.11 the same) after updating Google Play Services up to v8.1.0, i've got following exception:
java.lang.IncompatibleClassChangeError: The method 'void com.google.android.gms.common.api.GoogleApiClient.connect()' was expected to be of type interface but instead was found to be of type virtual (declaration of 'java.lang.reflect.ArtMethod' appears in /system/framework/core-libart.jar)
            at io.nlopez.smartlocation.location.providers.LocationGooglePlayServicesProvider.init(LocationGooglePlayServicesProvider.java:64)
            at io.nlopez.smartlocation.SmartLocation$LocationControl.<init>(SmartLocation.java:140)
            at io.nlopez.smartlocation.SmartLocation.location(SmartLocation.java:59)

**Comment by smer4ik** [comment]
The same error.

**Comment by ultraon** [comment]
As i understand correctly the cause of this problem is in following (https://developers.google.com/android/guides/releases):
```
GoogleApiClient, PendingResult, and OptionalPendingResult are now abstract classes instead of interfaces
```
And need only update Google Play Services version in SmartLocationLib, recompile it with new version number.

**Comment by biokys** [comment]
Waiting for it! ;-)

**Comment by mrmans0n** [comment]
This is fixed in version 3.2.0, which will be available in Maven Central today.

**Comment by janswist** [comment]
Error still occurs.
